### PR TITLE
Remove parent email section for LTI users

### PR DESCRIPTION
**Pull request by carl-codeorg**
Removes the parent email section from the finish signup page for LTI users:

Screenshot 2024-06-20 at 3 12 56 PM

## Links

[JIRA](https://codedotorg.atlassian.net/browse/P20-1000)

## Testing story

Tested locally to make sure no student email shows up

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by stephenliang** [comment]
FYI @hannahbergam

**Review by stephenliang — APPROVED** [review]
Would it be difficult to add a unit test that tests this? Something like `assert_not_select #parent-email-container`

**Review by nicklathe — APPROVED** [review]

File: dashboard/app/views/devise/registrations/_finish_sign_up.html.haml [hunk 1]
```diff
@@ -80,35 +80,36 @@
...
-              - if @user.errors[:parent_email_preference_opt_in].present?
-                %p.error= t('signup_form.email_preference_required')
-            .span1.parent-email-field-input{style:"display:block"}
-              .radio
-                = f.radio_button :parent_email_preference_opt_in, 'yes'
-                = f.label :parent_email_preference_opt_in, t('signup_form.email_preference_yes'), value: 'yes'
-              .radio
-                = f.radio_button :parent_email_preference_opt_in, 'no'
-                = f.label :parent_email_preference_opt_in, t('signup_form.email_preference_no'), value: 'no'
+    - unless is_lti
```
Comment (hannahbergam) [inline dashboard/app/views/devise/registrations/_finish_sign_up.html.haml:83]: I take it this line was the only *real change, and the rest is indentation?

**Review by hannahbergam — COMMENTED** [review]
I don't want to make any assumptions- were you able to confirm that you could still create accounts locally (and see the new users in local tables) in both an LTI and non-LTI scenario?

File: dashboard/app/views/devise/registrations/_finish_sign_up.html.haml (hunk 1 above)
Comment (carl-codeorg) [reply dashboard/app/views/devise/registrations/_finish_sign_up.html.haml:83]: Yep, sorry the diff isn't very clear. All this does is conditionally render the parent email section. And yes locally everything still works

Comment (hannahbergam) [reply dashboard/app/views/devise/registrations/_finish_sign_up.html.haml:83]: Awesome- thanks for confirming!

**Review by hannahbergam — APPROVED** [review]
LGTM!

**Pull request merged**